crates/trie/sparse/src/trie.rs

```diff
@@ -1217,6 +1229,9 @@ pub enum SparseNode {
         /// Pre-computed hash of the sparse node.
         /// Can be reused unless this trie path has been updated.
         hash: Option<B256>,
+        /// Pre-computed flag indicating whether the trie node should be stored in the database.
+        /// Can be reused unless this trie path has been updated.
+        store_in_db_trie: Option<bool>,
```

Comment:
unclear here how None differs from false?

Reply:
added a comment to all fields